hack/jenkins/job-configs/test-linkchecker.yaml

```diff
@@ -0,0 +1,66 @@
+- job-template:
```

Comment:
you probably also don't need to use a template; rather just specify the job directly. again see `kubernetes-verify-jenkins-jobs.yaml` for an example.